### DISCUSSION TITLE
Account suspension

### DISCUSSION THREAD

**Discussion by ghost** [community/community#33519] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello all, 
My github account: https://www.github.com/yashnerkar
has been abruptly suspended without any notice and intimidation. I request you to release my account of any suspension and atleast provide a valid reasoning for suspension so to avoid such situation next time.
This Github account has my whole academic  of code and I need it to show my projects to Companies.
Please do the needful as soon as possible. Thank You

**Comment by airtower-luna** [comment, marked as answer]
That's something you'll have to [contact GitHub support](https://support.github.com/request) about, us other users can't help there. Good luck!

**Reply by ghost** [reply]
I have already them, 4 days ago

**Reply by ghost** [reply]
But I didn't get any reply from them

**Reply by 3kh0** [reply]
@yash-png There is nothing else we can do here.

**Reply by queenofcorgis** [reply]
Hi there! Please be patient as GitHub Support works through tickets in the order they were received.

**Reply by ghost** [reply]
Okkay

**Reply by ijnrghjkdsmigywneig203** [reply]
> Okkay

What was the outcome, did you account end up getting reinstated?

**Reply by ghost** [reply]
> > Okkay
> 
> What was the outcome, did you account end up getting reinstated?

they wont reply. their support page is useless. best option is move away from github for better future.

**Reply by 3kh0** [reply]
@shekhawatanand Their support page is very helpful, if the account was suspended, there was a good reason for it. *Who would even use GitLab tbh*

**Reply by ghost** [reply]
> @shekhawatanand Their support page is very helpful, if the account was suspended, there was a good reason for it. _Who would even use GitLab tbh_

what is the use when they dont even reply in 20-25days? no value of our time? gitlab is good in terms of support.

**Reply by 3kh0** [reply]
@shekhawatanand I asked a question about code spaces and it got a response within a day. They don't reply to account suspensions, you should know the reason and you should have read through the ToS like a proper consumer of products. In the ToS it explains what repos you can create and not. Clearly @yash-png did not read through it and violated the terms, so that is their fault, and GitHub should not reinstate their account. GitHub rarely makes mistakes like these.

![Screenshot 2022-10-12 18 08 43](https://user-images.githubusercontent.com/58097612/195457336-1a5de33f-0f17-4ffc-ad40-f079fa0bd8e5.png)

**Reply by ghost** [reply]
> @shekhawatanand I asked a question about code spaces and it got a response within a day. They don't reply to account suspensions, you should know the reason and you should have read through the ToS like a proper consumer of products. In the ToS it explains what repos you can create and not. Clearly @yash-png did not read through it and violated the terms, so that is their fault, and GitHub should not reinstate their account. GitHub rarely makes mistakes like these.
> 
> ![Screenshot 2022-10-12 18 08 43](https://user-images.githubusercontent.com/58097612/195457336-1a5de33f-0f17-4ffc-ad40-f079fa0bd8e5.png)

most the users dont go through ToS thoroughly and make mistakes. Everyone make Mistakes, this doesnt mean they could suspend the account without warning, and giving no responses to the queries afterwards. they could have warned about violations or atleast they could give them some time to backup their important sources stored on github.

**Reply by 3kh0** [reply]
@shekhawatanand That is on them for not reading ToS. Big companies cannot make mistakes though, if someone was to make repos and start to flood them with copyrighted content or NSFW content, they have to take action or they will look bad, and they can not waste their time saying for 99% of users that the violated ToS, and they are not even required to provide a reason, as stated in their ToS.

![image](https://user-images.githubusercontent.com/58097612/195459662-038a26fc-50f3-4195-ae00-95f38ec3ebd2.png)

**Reply by DebiduttaPanigrahi** [reply]
My guthub account suddenly suspended automatically without any reason please recover my account fast i have multiple projects and important things present their please reply fast

**Comment by ijnrghjkdsmigywneig203** [comment]
Completely agree with this

**Reply by 3kh0** [reply]
What do you mean by "agree"?

**Comment by ijnrghjkdsmigywneig203** [comment]
I agree with what "shekhawatanand" said in the last message :)

**Comment by ijnrghjkdsmigywneig203** [comment]
No
